note.py

```diff
@@ -633,6 +643,7 @@ def onclose(self):
         """Saves the current note and closes the app."""
         try:
             self.noteframe.save()
+            self.__model.set_geometry(f"{self.root.winfo_width()}x{self.root.winfo_height()}")
```

Comment:
```suggestion
            self.__model.set_geometry(self.root.winfo_geometry())
```
maybe do not only save size of window, but complete geometry with its position?